hare/hare.go

```diff
@@ -5,6 +5,7 @@ import (
 	"github.com/spacemeshos/go-spacemesh/common/types"
 	"github.com/spacemeshos/go-spacemesh/hare/config"
 	"github.com/spacemeshos/go-spacemesh/log"
+	"go.uber.org/atomic"
```

Comment:
why not use the go stdlib atomic ?

Reply:
Changed to golangs atomic
please review the usage